.circleci/config.yml

```diff
@@ -331,7 +331,7 @@ jobs:
           name: Analyze Pull Request (pull-bot)
           command: |
             cd bots
-            DANGER_GITHUB_API_TOKEN="$PUBLIC_PULLBOT_GITHUB_TOKEN_A""$PUBLIC_PULLBOT_GITHUB_TOKEN_B" yarn danger ci --use-github-checks
+            DANGER_GITHUB_API_TOKEN="$PUBLIC_PULLBOT_GITHUB_TOKEN_A""$PUBLIC_PULLBOT_GITHUB_TOKEN_B" yarn danger ci --use-github-checks --failOnErrors
```

Comment:
> The ideal solution might be move this on GitHub Actions, that has a trigger for when the body is modified out of the box.

I think this is totally doable. It will also simplify the CircleCI and keep it focused on building only.

Reply:
I've added the workflow, I think now the result is much better.
It runs every time you update the body, and it runs in only 25 seconds.
(also we can use the GitHub Actions token)
<img width="918" alt="Screenshot 2022-05-19 at 16 56 00" src="https://user-images.githubusercontent.com/17830956/169327310-a27f09bc-a2ff-4a5d-b6d8-82f0c1e63e14.png">